src/css/_population-slider.scss

```diff
@@ -4,17 +4,14 @@
 // }
 
 .population-slider-container {
-  position: absolute;
   z-index: 1100;
   display: flex;
   flex-direction: column;
   align-items: center;
-  top: 10em;
-  left: 1em;
 }
 
 .population-slider-controls {
-  width: 360px;
+  width: 230px;
```

Comment:
I was about to change this to `100%`, but it's a larger refactor.
I can fix it in this PR, but I got halfway through and realized it would be easier after #241.